policies/voting-procedure.md

```diff
@@ -0,0 +1,71 @@
+The Public Voting Procedure of OTC
+==================================
+
+The following regulations complement the [OTC Voting Procedures] stated
+in the project bylaws. This policy affects only the public votes.
+
+[OTC Voting Procedures]: https://www.openssl.org/policies/omc-bylaws.html#otc-voting
+
+Vote Proposal
+-------------
+
+The votes are proposed in pull requests and issues of technical-policies
+repository on GitHub OpenSSL project.
+
+The vote regarding a policy change proposal is recorded directly in the
+pull request of the policy change proposal.
+
+Any other votes are recorded as separate issues in the repository.
+
+Casting the Votes
+-----------------
+
+The individual members of OTC cast their votes directly in the issue or
+pull request where the vote was proposed.
+
+When the votes are cast during an OTC meeting, the person responsible for
+taking the meeting minutes or the proposer of the vote copies the votes
+cast during the meeting into the issue.
+
+The Final Vote Records
+----------------------
+
+After the vote is closed the proposer of the vote records the final outcome
+and individual member votes in a separate file in the votes subdirectory of
+the repository.
```

Comment:
Are PRs required to add the vote files? Or can OTC members push the votes files directly after the close of a vote since they are implicitly approved by the act of voting.